packs/agents-of-edgewatch-bestiary/ofalth-zombie.json

```diff
@@ -425,9 +425,6 @@
                 {
                     "type": "paralyzed"
                 },
-                {
```

Comment:
I think this should be immune to poison instead.

Reply:
Good catch. That makes more sense for how it wound up at vitality then